content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md

```diff
@@ -41,6 +41,7 @@ When you sign up for an account, {% data variables.product.product_name %} provi
 {% ifversion ghec %}* Gravatar profile pictures are not supported with {% data variables.product.prodname_emus %}.{% endif %}
 
 {% endnote %}
+If your Gravatar image is associated with the email you use for {% data variables.product.product_name %}, it will be shown as your {% data variables.product.product_name %} profile picture by default (rather than an identicon). To change your {% data variables.product.product_name %} profile picture to your Gravatar image, you can either update your Gravatar account or upload a new image to{% data variables.product.product_name %}, which will override the Gravatar image.
```

Comment:
A few wording suggestions and adding some whitespace.

```suggestion

If you use Gravatar, and your Gravatar image is associated with the email you use for {% data variables.product.product_name %}, the image will be shown as your {% data variables.product.product_name %} profile picture by default (rather than an identicon). To change your {% data variables.product.product_name %} profile picture, you can either update your Gravatar account or upload a new image to {% data variables.product.product_name %}, which will override the Gravatar image.
```